The Applications page requires that every application runs in it's own cgroup
But this behaviour is only implemented in Plasma for Systemd systems.
So disable by default, since it is pretty much useless for us

--- src/systemmonitor.kcfg	2021-01-21 14:01:55.000000000 +0100
+++ -	2021-01-22 19:12:44.908000104 +0100
@@ -27,6 +27,7 @@
             <default>overview.page,applications.page,history.page,processes.page</default>
         </entry>
         <entry name="hiddenPages" type="StringList">
+            <default>applications.page</default>
         </entry>
     </group>
 </kcfg>
